tests/python/pants_test/util/test_objects.py

```diff
@@ -41,17 +39,24 @@ def test_none(self):
     with self.assertRaises(ValueError):
       SubclassesOf()
 
+  def test_str_and_repr(self):
+    superclasses_of_b = SuperclassesOf(self.B)
+    self.assertEqual("-B", str(superclasses_of_b))
```

Comment:
Yeah, I definitely prefer spelling this out in words; magic punctuation doesn't mean anything to me here... Let's just make `str` the same as `repr`

Reply:
I think there's room to make `str` more verbose than `repr`, but magic punctuation can leave.

Reply:
I have removed the magic punctuation!